### ISSUE TITLE
Add fallback icon on favicon image

### ISSUE THREAD

**Issue by Evavic44** [Evavic44/victoreke.com#95]
When the favicon link returns a missing image value, it throws an error and displays the alt text instead. Add a fallback icon that displays `onError`
